app/assets/stylesheets/scaffolds.scss

```diff
@@ -0,0 +1,65 @@
+body {
```

Comment:
I think this auto generated file should be deleted, because it messes up the styling.